InSituLib/VtkMappedMeshSource.cpp

```diff
@@ -151,6 +154,27 @@ int VtkMappedMeshSource::addIntProperty(vtkUnstructuredGrid &output,
 	return 1;
 }
 
+int VtkMappedMeshSource::addUnsignedProperty(vtkUnstructuredGrid &output,
+                                             MeshLib::Properties const& properties,
+                                             std::string const& prop_name) const
+{
+	boost::optional<MeshLib::PropertyVector<unsigned > const &> propertyVector(
+		properties.getPropertyVector<unsigned>(prop_name));
+	if(!propertyVector)
+		return 0;
+
+	vtkNew<VtkMappedPropertyVectorTemplate<unsigned> > dataArray;
+	dataArray->SetPropertyVector(const_cast<MeshLib::PropertyVector<unsigned> &>(*propertyVector));
+	dataArray->SetName(prop_name.c_str());
+
+	if(propertyVector->getMeshItemType() == MeshLib::MeshItemType::Node)
+		output.GetPointData()->AddArray(dataArray.GetPointer());
+	else if(propertyVector->getMeshItemType() == MeshLib::MeshItemType::Cell)
+		output.GetCellData()->AddArray(dataArray.GetPointer());
+
+	return 1;
+}
```

Comment:
The implementation of this method looks pretty much like the implementation of the int and double version. Maybe it is possible to generalize the implementation via a template?